blocks/form/form.js

```diff
@@ -169,6 +182,7 @@ function createSubmit(fd) {
     });
     obs.observe(wrapper);
   }
+  wrapper.appendChild(document.createElement('ul')); // for error messages
```

Comment:
Why do we need UL for displaying single error message?

Reply:
I am using the same css to display the error message, which we are using when we display any validation error in attachment component. UL can help show a list of messages, currently we have one(unsuccessful form submission or non 200) but in future if someone wants to show validation errors as well same ul can be used.

Reply:
> UL can help show a list of messages, currently we have one(unsuccessful form submission or non 200) but in future if someone wants to show validation errors as well same ul can be used.

I think we can update that to UL when required. Right now I would say we use a single div to show the error.

Reply:
ok, I will update the PR.